.github/workflows/ci.yml

```diff
@@ -6,8 +6,9 @@ jobs:
     strategy:
       matrix:
         go: [ 1.19, 1.18, 1.17, 1.16 ]
-    name: Tests Go ${{ matrix.go }}
-    runs-on: ubuntu-18.04
+        os: [ ubuntu-22.04, ubuntu-20.04 ]
+    name: Tests Go ${{ matrix.go }} # This name is used in main branch protection rules
+    runs-on: ${{ matrix.os }}
```

Comment:
I coped this from the main repo since the old ubuntu-18.04 is deprecated for golang tests now and thats why the actions never ran.